gum/backend-x86/gumstalker-x86.c

```diff
@@ -3470,6 +3459,91 @@ gum_exec_block_write_call_probe_code (GumExecBlock * block,
   }
 }
 
+#if GLIB_SIZEOF_VOID_P == 4 && !defined (HAVE_QNX)
+static void
+gum_exec_block_write_sysenter_continuation_code (GumExecBlock * block,
+                                     GumGeneratorContext * gc,
+                                     gpointer saved_ret_addr,
+                                     gpointer resolve_dynamically_label)
```

Comment:
The `resolve_dynamically_label` parameter is not needed – it should be moved here instead. A label is just a pointer value that's unique for a given `X86Writer` initialization.